src/gpu/locks.rs

```diff
@@ -0,0 +1,63 @@
+use fs2::FileExt;
+use log::info;
+use std::fs::File;
+
+const GPU_LOCK_NAME: &str = "/tmp/bellman.gpu.lock";
```

Comment:
This should probably use https://doc.rust-lang.org/std/env/fn.temp_dir.html instead of hard coding `/tmp`